src/ClassApplication/Controllers/ClassController.cs

```diff
@@ -40,11 +41,89 @@ public async Task<ActionResult<string>> CreateClassroom(Classroom classroom)
         /// <param name="id">classroom id</param>
         /// <returns>An array containing a value determined by the parameter</returns>
         [HttpGet("GetClassByID/{id}")]
-        public async Task<ActionResult<string>> GetClassroom(string id)
+        /*

```

Comment:
Just a little clean up here. could you remove these comment lines?

Reply:
Got It. Thank you very much.